alerta/auth/basic_ldap.py

```diff
@@ -16,6 +16,10 @@
 @auth.route('/auth/login', methods=['OPTIONS', 'POST'])
 @cross_origin(supports_credentials=True)
 def login():
+    # Allow LDAP server to use a self signed certificate
+    if current_app.config['LDAP_TLS_ALLOW_SELFSIGNED']:
```

Comment:
Can you please rename `LDAP_TLS_ALLOW_SELFSIGNED ` to `LDAP_ALLOW_SELF_SIGNED_CERT`?

Reply:
Renamed.